examples/java-matter-controller/java/src/com/matter/controller/commands/common/CommandManager.java

```diff
@@ -89,10 +89,10 @@ public final void run(String[] args) {
       }
     }
 
-    String[] temp = Arrays.copyOfRange(args, 1, args.length);
+    String[] temp = Arrays.copyOfRange(args, 2, args.length);
 
     try {
-      command.initArguments(args.length - 1, temp);
+      command.initArguments(args.length - 2, temp);

```

Comment:
```java
 command.initArguments(temp.length, temp);
 ```
 
 so you do not use a magic constant twive and decouple 'temp' from others. Also temp is a vague name. maybe `commandArguments` would be more readable.

Reply:
agree, done, thanks